src/pages/elements/User.js

```diff
@@ -2,7 +2,32 @@ import React from 'react';
 import Definition from '../../components/Definition';
 import Roles from './Roles';
 
+
+
 function User({ user }) {
+  var key;
+  if(user.public_gpg_key == null){
+    key = "-";
+  }
+  else if (user.public_gpg_key.length>= 50) {
+     key = <button onClick={downloadKey}> {user.public_gpg_key.substring(0,50) + '...'} </button>;
+  }
+  else {
+    key = <button onClick={downloadKey}> {user.public_gpg_key} </button>;
+  }
+  //taken from: https://stackoverflow.com/questions/44656610/download-a-string-as-txt-file-in-react
+  //there probably is a better way in react to do this 
+  function downloadKey(e) {
+    if(user.public_gpg_key != null){
+      e.preventDefault();
+      const element = document.createElement("a");
+      const file = new Blob([user.public_gpg_key], {type: 'text/plain'});
+      element.href = URL.createObjectURL(file);
+      element.download = `${user.redhat_username}_public_gpg_key.asc`;
```

Comment:
since the keys in app-interface are binary base64 encoded and not ascii armored, `.asc` is probably a bit misleading. i think that `.gpg` would make more sense. what do you think?